### ISSUE TITLE
Game Fix: None Available

### ISSUE THREAD

**Issue by Sequincer** [Soreepeong/XivAlexander#591]
Hi, despite using the latest XIVAlexander update, I can't see any game fixes.
![image](https://user-images.githubusercontent.com/96594510/224232339-f7263718-563e-4a55-9ca5-a2664f569a28.png)

**Comment by robyn0751** [comment]
I don't think this is relevant anymore, as this was for NIN that broke before 6.35. It's fixed now, but in case you're wondering how it'll show, go to Modding > TexTools Mod Packs > Show Dedicated Menu for it to appear. I installed .58 and this option was unchecked.
